deploy/cluster-monitoring-config-non-uwm/config.yaml

```diff
@@ -8,3 +8,6 @@ selectorSyncSet:
     - key: ext-managed.openshift.io/uwm-disabled
       operator: In
       values: ["true"]
+    - key: api.openshift.com/legal-entity-id
+      operator: NotIn
+      values: ${{SCALE_TEST_LEGALENTITY_IDS}}
```

Comment:
ORing on this means that his config will go out to all non-scale test clusters in every environment

Reply:
right, I asked about that. We can remove this OR but what does that mean for the non-uwm config since this is in prod today?

Do we need another split?